modules/apps/frontend-js/frontend-js-tooltip-support-web/src/main/resources/META-INF/resources/index.js

```diff
@@ -200,9 +199,15 @@ const TooltipProvider = () => {
 
 	useLayoutEffect(() => {
 		if (state.target && tooltipRef.current) {
-			setAlignment(
-				align(tooltipRef.current, state.target, POSITIONS.BottomCenter)
-			);
+			let position = POSITIONS.BottomCenter;
+
+			const dataAlign = state.target.dataset['tooltipAlign'];
+
+			if (dataAlign) {
+				position = ALIGN_POSITIONS_MAP[dataAlign];
+			}
```

Comment:
A "cute" idiom here to avoid the `let` binding is:

```js
const position =
	ALIGN_POSITIONS_MAP[state.target.dataset["tooltipAlign"]] ||
	POSITIONS.BottomCenter;
```